packages/react-native-editor/package.json

```diff
@@ -1,123 +1,122 @@
 {
-  "name": "@wordpress/react-native-editor",
-  "version": "1.30.0",
-  "description": "Mobile WordPress gutenberg editor.",
-  "author": "The WordPress Contributors",
-  "license": "GPL-2.0-or-later",
-  "keywords": [
-    "wordpress",
-    "react-native"
-  ],
-  "private": true,
-  "config": {
-    "jsfiles": "./*.js src/*.js src/**/*.js src/**/**/*.js",
-    "scssfiles": "src/*.scss src/**/*.scss"
-  },
-  "homepage": "https://github.com/WordPress/gutenberg/tree/master/packages/react-native-editor/README.md",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/WordPress/gutenberg.git",
-    "directory": "packages/react-native-editor"
-  },
-  "bugs": {
-    "url": "https://github.com/WordPress/gutenberg/issues"
-  },
-  "engines": {
-    "node": ">=10",
-    "npm": ">=6.9"
-  },
-  "main": "src/index.js",
-  "react-native": "src/index",
-  "dependencies": {
-    "@babel/runtime": "^7.9.2",
-    "@react-native-community/slider": "git+https://github.com/wordpress-mobile/react-native-slider.git#5ad284d92b8d886e366445bf215be741ed53ddc6",
-    "@wordpress/api-fetch": "file:../api-fetch",
-    "@wordpress/block-editor": "file:../block-editor",
-    "@wordpress/block-library": "file:../block-library",
-    "@wordpress/blocks": "file:../blocks",
-    "@wordpress/components": "file:../components",
-    "@wordpress/data": "file:../data",
-    "@wordpress/edit-post": "file:../edit-post",
-    "@wordpress/element": "file:../element",
-    "@wordpress/hooks": "file:../hooks",
-    "@wordpress/i18n": "file:../i18n",
-    "@wordpress/react-native-aztec": "file:../react-native-aztec",
-    "@wordpress/react-native-bridge": "file:../react-native-bridge",
-    "fast-average-color": "^4.3.0",
-    "jed": "^1.1.1",
-    "jsc-android": "^241213.1.0",
```

Comment:
`jsc-android` isn't explicitly used in the package. It's listed as a direct dependency of `react-native` though.